README.md

```diff
@@ -6,7 +6,7 @@ Quick links:
 - [Examples of use](#usage)
 - [Configuration](#config)
 - [How to get latest version](#getit)
-- API Docs [Core](https://oss.sonatype.org/service/local/repositories/releases/archive/com/ccadllc/cedi/dtrace-core_2.12/1.4.0/dtrace-core_2.12-1.4.0-javadoc.jar/!/com/ccadllc/cedi/dtrace/index.html) [Logging](https://oss.sonatype.org/service/local/repositories/releases/archive/com/ccadllc/cedi/dtrace-logging_2.12/1.4.0/dtrace-logging_2.12-1.4.0-javadoc.jar/!/com/ccadllc/cedi/dtrace/logging/index.html) [Logstash](https://oss.sonatype.org/service/local/repositories/releases/archive/com/ccadllc/cedi/dtrace-logstash_2.12/1.4.0/dtrace-logstash_2.12-1.4.0-javadoc.jar/!/com/ccadllc/cedi/dtrace/logstash/index.html)
```

Comment:
Intentional?

Reply:
sigh.  yes.  And money.  and http4s.  and xb3.